### PR TITLE
Get a user's chat handles

### PR DESCRIPTION
**Pull request by jsteiner**

### CODE REVIEW

File: lib/cog_api/client.ex [hunk 1]
```diff
@@ -83,4 +83,5 @@ defmodule CogApi.Client do
   @callback user_create(%Endpoint{}, %{}) :: {atom, %User{}}
   @callback user_update(%Endpoint{}, String.t, %{}) :: {atom, %User{}}
   @callback user_delete(%Endpoint{}, String.t) :: atom
+  @callback user_chat_handles(%Endpoint{}, String.t) :: {:ok, [%ChatHandle{}]}
```
Comment (drapergeek) [inline lib/cog_api/client.ex]: I know that a user is required for a chat handle `show` action but it seems like this belongs on the `ChatHandle` module instead.

I think a good principle (which I've only recently decided on) is that every function in a module should return the resource of the same name. `User` should only return `%User{}`. Thoughts?

**Comment by drapergeek** [comment]
Other than my opinions on where this functionality goes, this LGTM

File: lib/cog_api/client.ex (hunk 1 above)
Comment (jsteiner) [inline lib/cog_api/client.ex]: Yeah, I dig that. `chat_handles_for_user(user_id, endpoint)`?

Comment (drapergeek) [inline lib/cog_api/client.ex]: I think so

**Pull request merged**